app/plugin_install.go

```diff
@@ -199,8 +198,7 @@ func (ch *Channels) InstallMarketplacePlugin(request *model.InstallMarketplacePl
 	if prepackagedPlugin != nil {
 		fileReader, err := os.Open(prepackagedPlugin.Path)
 		if err != nil {
-			err = errors.Wrapf(err, "failed to open prepackaged plugin %s", prepackagedPlugin.Path)
-			return nil, model.NewAppError("InstallMarketplacePlugin", "app.plugin.install_marketplace_plugin.app_error", nil, err.Error(), http.StatusInternalServerError)
+			return nil, model.NewAppError("InstallMarketplacePlugin", "app.plugin.install_marketplace_plugin.app_error", nil, fmt.Sprintf("failed to open prepackaged plugin %s: %s", prepackagedPlugin.Path, err.Error()), http.StatusInternalServerError)
```

Comment:
0/5, just curious for this change. Was this just a refactoring, or was there an error in the code?

Reply:
This is just a refactoring. I think this is not an idiomatic way of using error wraps.